docs/plugins/find_and_install_plugin.md

```diff
@@ -23,7 +23,7 @@ directly from within napari:
 
 1. From the “Plugins” menu, select “Install/Uninstall Plugins...”.
 
-   ![napari viewer's Plugins menu with Install/Uninstall Plugins as thr first item.](/images/plugin-menu.png)
+   ![napari viewer's Plugins menu with Install/Uninstall Plugins as thr first item. ](/images/plugin-menu.png)
 
 2. In the resulting window that opens, where it says “Install by name/URL”,
```

Comment:
```suggestion
2. In the Plugin dialog that opens, where it says “Install by name/URL”,
```

Updating the in-paragraph text for consistency with the image description - I think referring to it as Plugin dialog is more specific anyway.